rfcs/text/0000-project-completion-stage.md

```diff
@@ -0,0 +1,74 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Project completion stage RFC
+
+## Summary
+
+This RFC proposes a process of releasing a contribution, or project, only after it has been completed by the DCP in order to maximize the value to the data consumer through reducing user confusion and minimizing errors in the final product.
+
+## Author(s)
+
+ [Jason Hilton](mailto:jahilton@stanford.edu)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Definitions
+
+A contribution, or project, is considered to be **version-complete** when the contributor confirms (by way of the wrangler) that they have provided all of the metadata and raw data that they plan to submit at the current time, the wrangler submits those to ingest, and the DCP has delivered every possible product, given current DCP capabilities.
+
+## Motivation
+
+**Display only completed data:**
+Any project that contains raw data with a corresponding analysis pipeline should be considered in progress until all downstream analysis is version-complete, validated, and available. Currently, a contributed project is made accessible to the public as soon as raw data & metadata are submitted. The trickling in of data can lead to data consumer confusion as to if/when more data will be added to the project. The proposed solution allows the DCP to complete the version of the project with DCP outputs before the project becomes publicly accessible.
+
+**More thorough data validation:**
+The downstream services of the DCP have the potential to uncover metadata and data errors. Viewing the version-complete project as the user will view it can also identify errors. A solution to this is a setting on projects allowing signed-in DCP members to access the data in order to perform a final ‘sanity check’.
```

Comment:
This, I am very concerned that this will unduly slow down making data available to the public if we put too many humans in the way. Seeking to automate as much checking and accepting that we will never catch 100% of errors but we can get better overtime seems like the best compromise approach

Reply:
Fully agree on all of this. Our goal is to improve, not catch _everything_.
We set the review period at 1 week max knowing that it will likely be fully manual to begin. But the goal will be to automate the DataOps checklist/validation procedure as soon as possible.

Reply:
I am all for automating the QA for data access, we need it. This is clear in recent months. curation by QC is something that makes sense for distributions but there should always be ways of accessing data that passed QA but not QC. That data is valuable to some use cases and may be recovered by clever people.

Reply:
I think we're all in agreement here. Our goal is to get proper QA checks in place first, and then we'll iteratively work at automating more and more to be more efficient.
Resolving this thread, please renew comments if I can address things further in the RFC.